src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx

```diff
@@ -33,119 +33,188 @@ redirects:
   - /docs/mobile-monitoring/new-relic-mobile-android/legacy/installing-android-apps-eclipse
   - /docs/mobile-monitoring/new-relic-mobile-android/install-configure/installing-android-apps-maven
   - /docs/mobile-monitoring/new-relic-mobile-android/install-configure
+  - /docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-android-gradle-android-studio
 ---
 
+You can install the mobile agent for Android monitoring with our [guided install](https://onenr.io/0kjnEoryzRo), or you can install manually. The guided install is a good option for simple app environments. For other cases, you can follow the steps below to install the Android agent.
```

Comment:
I understand that Android studio is a development environment. There's a second one for Android apps -- do we support that one as well? Are there any stopgaps we can offer users with Android apps using different environments?